jobbergate-api/jobbergate_api/cli/db.py

```diff
@@ -0,0 +1,55 @@
+import subprocess
+
+import docker_gadgets
+import typer
+
+from jobbergate_api.config import settings
+from jobbergate_api.storage import build_db_url
+
+app = typer.Typer()
+
+
+@app.command()
+def login(test: bool = typer.Option(False, help="Log into the test database instead"),):
+    """
+    Logs into a local database.
+    """
+    subprocess.run(["pgcli", build_db_url(force_test=test)])
+
+
+@app.command()
+def start_dev():
+    """
+    Starts a local postgres database for local development.
+    """
+    docker_gadgets.start_service(
+        "dev-jobbergate-postgres",
+        image="postgres:14.1",
+        env=dict(
+            POSTGRES_PASSWORD=settings.DATABASE_PSWD,
+            POSTGRES_DATABASE=settings.DATABASE_NAME,
+            POSTGRES_USER=settings.DATABASE_USER,
+        ),
+        ports={"5432/tcp": settings.DATABASE_PORT},
+    )
+
+
+@app.command()
+def start_test():
+    docker_gadgets.start_service(
+        "test-jobbergate-postgres",
+        image="postgres:14.1",
+        env=dict(
+            POSTGRES_PASSWORD=settings.TEST_DATABASE_PSWD,
+            POSTGRES_DATABASE=settings.TEST_DATABASE_NAME,
+            POSTGRES_USER=settings.TEST_DATABASE_USER,
+        ),
+        ports={"5432/tcp": settings.TEST_DATABASE_PORT},
+        tmpfs={"/var/lib/postgresql/data": "rw"},
+    )
+
+
+@app.command()
+def start_all():
+    start_dev()
+    start_test()
```

Comment:
is this CLI app really useful? I just see complexity being added to the project and more code to maintain in the long run, don't you?

Reply:
No, these commands in the cli are extremely useful. For example, you can run `db login` and log directly into the database with the settings you have configured for the app. This is very helpful when debugging data issues. Othewise, you have to:
* remember the command line to log into postgres
* fetch and copy all the values from your environment to do so.

For the `start` commands, these are nice if you want to quickly spin up a postgres database in docker for running unit tests and such.

These commands are converted from previous API projects I've worked on, and they proved very useful in development for myself and other devs on the team. I would like to keep them.

These commands are not used in production, they are only for helping developers. There is very little maintenance required here.